tests/integration/src/test/resources/org/mule/test/integration/xml/xslt-functional-test-service.xml

```diff
@@ -17,7 +17,7 @@
     <!--// START SNIPPET: transformer-config -->
     <mulexml:xslt-transformer name="xslt">
         <mulexml:xslt-text>
-            <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+            <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns="">
```

Comment:
Same as previous comment. Here is the service configuration for the same junit integration test.